src/Processes/ChurnProcess.php

```diff
@@ -43,7 +43,7 @@ public function __construct(File $file, Process $process, string $type)
      * Start the process.
      * @return void
      */
-    public function start()
+    public function start(): void
```

Comment:
All of the changes to method signatures of `public`, as well as `protected` methods of non-`final` classes should be considered as backwards-incompatible changes, I believe.